### DISCUSSION TITLE
Deleting Files

### DISCUSSION THREAD

**Discussion by njrusty** [JVital2013/vitality-goes#81] [General]
Is there a way to automatically have all the files deleted, say every other day, 2 days, 3 days...etc??

**Comment by stonegrift** [comment]
There are scripts available in the scripts folder that will clean up some of the files. Such as compressing all the Emwin text files and deleting old images.  They work fairly well, if you are somewhat skilled with installing the supporting software.

**Reply by njrusty** [reply]
Appreciate the feedback. I see the necessary scripts are already installed onto my system. And of course the frustrating part (based on the documentation) is to copy the scriptconfig.ini to the scripts folder. Cannot find this .ini file anywhere. I wish the documentation would show (for non-linux savy people...where such files are located). Not a knock against the programmer, just not everyone are linux pros.

**Reply by stonegrift** [reply]
scriptconfig.ini is in the configs folders it should have been copied over when you placed your ini files.

**Reply by njrusty** [reply]
Anyone have an example scriptconf.ini file they are willing to share???  Seeing examples always helps configuring files to each own system.

**Reply by stonegrift** [reply]
Let me see if I can annotate the scriptconfig.ini file here since mine is working to some extent.
videoDir is where you want the videos stored after they have been rendered. Best place for the Vitality goes page is in the Apache web page server in this case it's "/var/www/html/videos"  I suppose you could place it else where but then you run up against permissions and ownership.
abiSrcDir is the path to your saved Goes content.  Both GoesTools and Satdump save in known file structures. And connecting it here allows for a simpler shortcut in the next variable.
abiImgSource  and abiVidName are linked in that they need to have the same number of entries. And you can change them.

abiImgSource uses the abiVidName are a pair of single column arrays. (think simple list) that you are lining up entries on.

In abiImgSource you are setting the path to the image you want to animate. and abiVidName is the name of the saved video
the first video that is to be made is the Full disc /full color image.  In my case it's "$abiSrcDir/goes18/fd/fc"
That is the path to the images that have been saved.  This uses the "$abiSrcDir/goes18/fd/fc" rather than having to type out /home/YourUserName/GOESdirectory/fd/fc"  which in my case is "$abiSrcDir/goes18/fd/fc" rather than having to type /home/Warren/goes/goes18/fd/fc" Much shorter this way.  And the video name saved in abiVidName is "GOES18FalseColor"  you place each path and video name in the same order.  If you mess this part up your might be set the Goes18 full disk full color still image and the video might be from GOES18ch2.  For this part I've found it almost works better to make a spreadsheet listing each video path and the hoped for video name. in a second column.   
By default there are a lot of videos being made but they aren't turned on in the abi.ini file. They are turned off by being commented  out with a semi-colon. (;)

That takes us down to the emwin content.  this part is the same as for the ABI stuff we did before except these are for the emin images.  This you can browse through the emwin.ini file for the radars you're interested in maiking movies for.  This is a bit different in that you're looking for specific radar names. like RADNTHES is the radar area for the Northeast and was called "LocalRadar" for the file name.  I actually added an additional one here since I live in the Pacific Northwest RADPACNW for emwinCodeName and  RadPACNW for the mp4 file name added on the end of emwinVideoName

The next section which is information for the Sanchez graphics tool.  And you can see how it goes with the source directories and destination directories.  There is a specific script in Sanchez that combines the two satellite images from Goes 16 and 18 in a composite image.  It needs a place to stash the images. I just had it point to a new folder in the goes storage directory called composite. 
The last entry is the path to Sanchez.  So it's normally your user folder/sanchez with it's version number.  I had to compile it on my own to get it to work and mine ended up being "sanchezPath=/home/Warren/sanchez/output/Sanchez"  That last Snachez is the actual executable file.   The only issue I have with the script is that in the emwin directory I have yet to receive any Himawari images through emwin so it complains when it tries to make a video of it.  The Himawari abi vidoes do fine.
